src/Plugin/Select/Handler.php

```diff
@@ -184,11 +207,12 @@ protected static function handleSelectFromTables(Client $manticoreClient, Payloa
 				foreach ($createTables as $createTable) {
 					$row = static::parseTableSchema($createTable);
 					$data[$i] = [];
+					self::unifyFieldNames($payload->fields);
 					foreach ($payload->fields as $field) {
 						[$type, $value] = static::TABLES_FIELD_MAP[$field]
-							?? static::TABLES_FIELD_MAP[strtolower($field)] ?? ['field', $field];
+							?? static::TABLES_FIELD_MAP[strtolower($field)] ?? ['field', strtolower($field)];
 						$data[$i][$field] = match ($type) {
-							'field' => $row[$value],
+							'field' => $row[$value] ?? (static::DEFAULT_FIELD_VALUES[$field] ?? null),
```

Comment:
Just to be sure we use "$value" and "$field", is it normal? Or just missprint.

Reply:
Yes, it's what is intended - defining a row value by a field name.